docs/django-apps.md

```diff
@@ -0,0 +1,67 @@
+# Using Caluma as django apps
+
+## Disclaimer
+
+It is highly recommended to use Caluma as a dedicated service. However, there are usecases, where
+it might make sense to integrate Caluma into another django project.
+
+If you just want to get Caluma up and running, please see the documentation about [setting up
+the Caluma service](configuration.md).
```

Comment:
Add a disclaimer that Caluma only works with PostgreSQL. Direct integration won't work on other databases.

Reply:
```suggestion
the Caluma service](configuration.md).

Please beware that Caluma only works with PostgreSQL, and requires the `psqlextra` backend to use the advanced features (such as JSON fields etc)
```